README.md

```diff
@@ -290,7 +301,7 @@ You can use `withConditions` in conjunction with `forRoute` to set strict condit
 ```jsx
 <Fragment forRoute='/'>
   <div>
-    <Fragment forRoute='/home'><Home /></Fragment>
+    <Fragment forRoute='/'><Home /></Fragment>
```

Comment:
This seems like a more common pattern, and was the first confusion I ran into when nesting routes (since they're then redundant).